src/Standards/Generic/Docs/Commenting/DocCommentStandard.xml

```diff
@@ -0,0 +1,268 @@
+ * Short description.
+<em> *</em>
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The DocBlock must have a short description, and it must be on the first line.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: DocBlock with a short description on the first line.">
+        <![CDATA[
+/**
+ * <em>Short description.</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: DocBlock without a short description or short description not on the first line.">
+        <![CDATA[
+/**
+ * <em></em>@return int
+ */
+
+/**
+<em> *</em>
+ * Short description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    The short and long descriptions must start with a capital letter.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Short and long descriptions start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>S</em>hort description.
+ *
+ * <em>L</em>ong description.
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Short or long descriptions don't start with a capital letter.">
+        <![CDATA[
+/**
+ * <em>s</em>hort description.
+ *
+ * <em>l</em>ong description.
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There must be exactly one blank line separating the short description, the long description and tag groups.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: One blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *<em>
+ * Long description.
+<em> *</em>
+ * @param int $foo
+ */
+        ]]>
+        </code>
+        <code title="Invalid: More than one or no blank line separating the short description, the long description and tag groups.">
+        <![CDATA[
+/**
+ * Short description.
+<em> *
+ *
+</em>
+ * Long description.
+ * @param int $foo
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must be grouped together.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+<em> * @param int $foo
+ * @param string $bar</em>
+ */
+        ]]>
+        </code>
+        <code title="Invalid: Parameter tags not grouped together.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+<em> *</em>
+ * @param string $bar
+ */
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    Parameter tags must not be grouped together with other tags.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Parameter tags are not grouped together with other tags.">
+        <![CDATA[
+/**
+ * Short description.
+ *
+ * @param int $foo
+ *
+<em> * @author Some Author
+ * @return int</em>
+ */
```

Comment:
As the rule is about the param tags, would it make sense for the `<em>` highlighting to wrap the `@param` tag(s) instead of the non-`@param` tags ?

Reply:
I agree and I implemented it in dc22715983ac5a6c76ded1fd5e79e823f619f254